python/tests/test_highlevel.py

```diff
@@ -653,6 +654,13 @@ def test_edge_diffs(self):
         for ts in get_example_tree_sequences():
             self.verify_edge_diffs(ts)
 
+    def test_edge_diffs_names(self, simple_degree2_ts_fixture):
```

Comment:
This tests the item getter of the new named tuple, but we've lost the test of nested destructuring that was `for (left, right), edges_out, edges_in in ts.edge_diffs():`

Reply:
Oh yes. Good point. I've reinstated the old test scheme. I guess together with the new one that's covered now.